autogpt_platform/backend/backend/data/execution.py

```diff
@@ -69,10 +69,55 @@ class GraphExecutionMeta(BaseDbModel):
     ended_at: datetime
 
     class Stats(BaseModel):
-        cost: int = Field(..., description="Execution cost (cents)")
-        duration: float = Field(..., description="Seconds from start to end of run")
-        node_exec_time: float = Field(..., description="Seconds of total node runtime")
-        node_exec_count: int = Field(..., description="Number of node executions")
+        model_config = ConfigDict(
+            extra="allow",
+            arbitrary_types_allowed=True,
+        )
+
+        cost: int = Field(
+            default=0,
+            description="Execution cost (cents)",
+        )
+        duration: float = Field(
+            default=0,
+            description="Seconds from start to end of run",
+        )
+        duration_cpu: float = Field(
+            default=0,
+            description="CPU sec of duration",
+        )
+        node_exec_time: float = Field(
+            default=0,
+            description="Seconds of total node runtime",
+        )
+        node_exec_time_cpu: float = Field(
+            default=0,
+            description="CPU sec of node_exec_time",
+        )
+        node_exec_count: int = Field(
+            default=0,
+            description="Number of node executions",
+        )
+        node_error_count: int = Field(
+            default=0,
+            description="Number of node errors",
+        )
+        error: str | None = Field(
+            default=None,
+            description="Error message if any",
+        )
```

Comment:
Please also add the new fields to `GraphExecutionMeta.stats` here:
https://github.com/Significant-Gravitas/AutoGPT/blob/1ad6c76f9ccf9f356eb742f9f9c6d99ab14fb2d4/autogpt_platform/frontend/src/lib/autogpt-server-api/types.ts#L272-L288

Reply:
Added